src/kerodon/impl.clj

```diff
@@ -86,12 +86,14 @@
     link
     (not-found "link" selector)))
 
-(defn- field-to-selector [elem]
-  (if-let [id (get-in elem [:attrs :id])]
-    (form-element-by {:id id})
-    (if (= "radio" (get-in elem [:attrs :type]))
-      (form-element-by (select-keys (:attrs elem) [:name :value]))
-      (form-element-by {:name (get-in elem [:attrs :name])}))))
+(defn- field-to-selector [elem & [selector]]
+  (if selector
```

Comment:
I think you could put this condition in `form-element-selector` instead, as I think it only ever kicks in when called from there.